### DISCUSSION TITLE
Welcome to django-tenants Discussions!

### DISCUSSION THREAD

**Discussion by tomturner** [django-tenants/django-tenants#535] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asixela** [comment]
Hi,
I'm a  french python developper, I'm working on an project using django 2.2 and django-tenant-schemas (https://github.com/bernardopires/django-tenant-schemas). I would like to upgrade to django 3.0/3.1, but it seems that django-tenant-schemas does not support django 3.0 and is not maintained since 2019. So I would like to change and use django-tenants. Could you please, explain to me how to switch from django-tenant-schemas to  django-tenants?

**Reply by tomturner** [reply]
Can you copy this into https://github.com/django-tenants/django-tenants/discussions/537

**Reply by tomturner** [reply]
I have added what I would do.

**Reply by asixela** [reply]
Thank you, that's pretty much what I had identified to be done.

**Comment by itzomen** [comment]
Great to be here, I'm a Django developer and would love to contribute. I got started with django-tenants but noticed its not doesn't support Django 3.2 yet

**Comment by tomturner** [comment]
It does support 3.2 I just haven’t released it to pypi yet

**Reply by itzomen** [reply]
Alright, thanks for the reply
